apps/src/gamelab/animationLibrary.json

```diff
@@ -350,16 +350,15 @@
     "bronze_1": {
       "name": "bronze_1",
       "aliases": [
-        "jumper",
```

Comment:
aliases seem like a great thing to go back to and add. For now since we know what the kenny images are, it shouldn't be too hard to levelbuilders to find them.